client/state/index.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import thunkMiddleware from 'redux-thunk';
+import { createStore, applyMiddleware, combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import sharing from './sharing/reducer';
+import sites from './sites/reducer';
+import ui from './ui/reducer';
+
+/**
+ * Module variables
+ */
+const reducer = combineReducers( {
+	sharing,
+	sites,
+	ui
+} );
+
+export function createReduxStore() {
```

Comment:
A more practical example: If a component at an arbitrary point in the render hierarchy needs to display a notice, it's a good opportunity to use `react-redux` `connect`'s second argument, [`mapDispatchToProps`](https://github.com/rackt/react-redux/blob/master/docs/api.md#connectmapstatetoprops-mapdispatchtoprops-mergeprops-options).

```jsx
import React, { Component } from 'react';
import { bindActionCreators } from 'redux';
import { connect } from 'react-redux';
import { newNotice } from 'state/notices/actions';

class MyComponent extends Component {
	render() {
		return (
			<button onClick={ this.props.showNotice }>
				Show Notice
			</button>
		);
	}
}

export default connect( null, ( dispatch ) => {
	return bindActionCreators( {
		showNotice: newNotice
	}, dispatch );
} )( MyComponent );
```